OfficeMicroService/Application/Services/DTO/OfficeForChangeDTO.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson;
+using System.ComponentModel.DataAnnotations;
+
+namespace OfficeMicroService.Application.Services.DTO
+{
+    public class OfficeForChangeDTO
+    {
+        private const string NumberRegex = "[+]{1}[0-9]{12}";
+
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string PhotoId { get; set; }
+        [BsonRequired]
```

Comment:
It's not possible to check this with ModelState required so I was able to create an entity with all field equal to null, please fix. Entity is is 63a084b7dd56614723d3c51a